### ISSUE TITLE
Changelog is not updated with new releases

### ISSUE THREAD

**Issue by chimon2000** [Renesanse/osam#16]
It's currently impossible to know what changed without looking at the commit history.

**Comment by Renesanse** [comment]
you are right, I will create a separate branch for experimental features. I didn't know people use my lib in not pet projects. My bad, sorry.
